lib/install/proto/installer.proto

```diff
@@ -74,6 +74,9 @@ message AbortRequest {
 message ShutdownRequest {
     // Completed indicates that the operation has been successfully completed
     bool completed = 1;
+    // ExitCode optionally specifies the exit code for the service.
+    // 0 value is considered success
+    int32 exitCode = 2;
```

Comment:
Nit on the comment, 0 is success but was is code 2?

Reply:
it's a protobuf tag value, not the value for the exit code if I understood your comment correctly.

Reply:
My bad.